.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Question
+    url: https://github.com/zowe/imperative/discussions
```

Comment:
should we take them directly to the "Q&A" section?
Here is a direct link:
- https://github.com/zowe/imperative/discussions/new?category=q-a

Reply:
I'm thinking that first viewing any other QAs might help prevent duplicate threads, if someone sees their question was already asked they might just jump into that discussion versus starting a new one. Thoughts? @awharn @t1m0thyj @zFernand0

Reply:
oh whoops just realizing now that I didn't have the right link to questions page sorry